platform-core-module/src/main/java/com/softicar/platform/core/module/access/module/instance/ModuleInstancePage.java

```diff
@@ -26,14 +26,14 @@ public EmfPagePath getPagePath(EmfPagePath modulePath) {
 	}
 
 	@Override
-	protected IEmfTable<?, ?, SystemModuleInstance> getTable() {
+	protected IEmfTable<?, ?, AGCoreModuleInstance> getTable() {
 
 		return AGModuleInstance.TABLE;
 	}
 
 	// FIXME: remove once the scope is not SystemModuleInstance
```

Comment:
so... what about this?

Reply:
I just removed the FIXME but left the code unchanged. We will come back here, when we remove the `CorePermissions.SUPER_USER`.